recipes/pyclesperanto-prototype/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pyclesperanto-prototype" %}
+{% set version = "0.12.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyclesperanto_prototype-{{ version }}.tar.gz
+  sha256: 6eda7a426c00bc280ac2f355c740c40dca442f7c3c507e48d4e5d1565985263d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - matplotlib-base
+    - numpy !=1.19.4
+    - platformdirs
+    - pyopencl
+    - python >=3.6
+    - scikit-image >=0.18.0
+    - toolz
+    - transforms3d
+
+test:
+  imports:
+    - pyclesperanto_prototype
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+    - pocl  # [not win]
```

Comment:
Do we have an opencl driver for win so the import check does not fail @jaimergp?

On https://documen.tician.de/pyopencl/misc.html:

> to install a CPU-based OpenCL driver. On Windows, you may install e.g. the [CPU OpenCL driver from Intel](https://software.intel.com/en-us/articles/opencl-drivers#latest_CPU_runtime). On macOS, pocl can offer a marked robustness (and, sometimes, performance) improvement over the OpenCL drivers built into the operating system.

Reply:
I'm not sure if this will help here, but @StRigaud made a very similar project build on github CI. Maybe, this config helps?
https://github.com/clEsperanto/CLIc_prototype/blob/master/.github/workflows/build.yml#L22-L28

Furthermore, pyclesperanto is build on pyopencl, which is available on conda-forge. Both should have very similar dependencies. Maybe that receipe can help, and/or @inducer can give hints?
https://github.com/conda-forge/pyopencl-feedstock

Reply:
Maybe https://anaconda.org/conda-forge/oclgrind @goanpeca ?